ctapipe/image/geometry_converter_hex.py

```diff
@@ -397,8 +397,8 @@ def convert_geometry_hex1d_to_rect2d(geom, signal, key=None, add_rot=0):
         new_geom = CameraGeometry(
             cam_id=geom.cam_id + "_rect",
             pix_id=ids,  # this is a list of all the valid coordinate pairs now
-            pix_x=grid_x * u.meter,
-            pix_y=grid_y * u.meter,
+            pix_x=u.Quantity(grid_x.ravel(),  u.meter),
+            pix_y=u.Quantity(grid_y.ravel(),  u.meter),
```

Comment:
will this break any of the code from the machine-learning studies who later write this out as a 2D image? Perhaps we should at least store the original dimensions as metadata in the case the original  geometry was 2D. (I agree we shouldn't normally support 2D though, but having the metadata would allow one to `reshape` back to 2D later)

Reply:
to be more clear, the main use case of the GeometryConverter code is to produce 2D images that can be fed to image processing routines, so we need to support that somehow.

Reply:
> will this break any of the code from the machine-learning studies who later write this out as a 2D image?

I have no clue. All the tests pass. I did not change any test code concerning the geometry converters.

Reply:
Also, I think most of the behaviour was broken before.

If there was a camera roation set, but pxiel shape was not 1d, it was silently ignored, no warning, no error.

Reply:
>I have no clue. All the tests pass. I did not change any test code concerning the geometry converters.
I guess we need to look at the other projects that use this and see what test should be added...

Reply:
Or just wait until people expecting different behaviour open issues or PRs.

Reply:
I think we cannot start guessing what people might want.